go.mod

```diff
@@ -170,6 +170,7 @@ require (
 
 //These libs are used to allow orb extend opentelemetry features
 require (
+	github.com/google/uuid v1.3.0
```

Comment:
not needed i think?

Reply:
Yeah, I added because I added some span IDs to track each concurrent request, will remove it.